tests/scripts/thread-cert/backbone/test_mlr_multicast_routing_across_thread_pans.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2020, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS 'AS IS'
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+# This test verifies that the basic MLR feature works.
+#
+import unittest
+
+import config
+import thread_cert
+from pktverify.packet_verifier import PacketVerifier
+
+CH1 = 11
+CH2 = 22
+
+PBBR1 = 1
+ROUTER1 = 2
+PBBR2 = 3
+ROUTER2 = 4
+HOST = 5
+
+MA1 = 'ff04::1234:777a:1'
+
+BBR_REGISTRATION_JITTER = 1
+WAIT_REDUNDANCE = 3
+
+
+class TestMlr(thread_cert.TestCase):
+    USE_MESSAGE_FACTORY = False
+
+    # Topology:
+    #   --------(eth)-----------
+    #       |     |      |
+    #     PBBR  HOST   PBBR2
+    #      |            |
+    #    ROUTER1      ROUTER2
+    #
+    TOPOLOGY = {
+        PBBR1: {
+            'name': 'PBBR1',
+            'allowlist': [ROUTER1],
+            'is_otbr': True,
+            'version': '1.2',
+            'router_selection_jitter': 1,
+            'bbr_registration_jitter': BBR_REGISTRATION_JITTER,
+            'channel': CH1,
+            'prefer_router_id': PBBR1,  # Use prefer_router_id to avoid Router ID conflicts in the two channels.
```

Comment:
Is it necessary? Any issues if not?

Reply:
In real world, same short addresses in different partitions should be allowed. 

Not sure if the failure you met is specific for simulation, fine to me to keep it as is if yes.
But I would suggest to remove the prefer_router_id configuration and figure out the root cause for the failure when it occurs, just in case there is some real issue which should be fixed.

Thoughts?

Reply:
removed.